<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_radar_config</class>
 <widget class="QDialog" name="set_radar_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set_Config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/set_radar_config.png</normaloff>:/images/set_radar_config.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Relay control message (0x8) is sent if true and the&lt;/p&gt;&lt;p&gt;collision detection is activated&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="StoreInNVM">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stores the current configuration to non-volatile&lt;/p&gt;&lt;p&gt;memory to be read and set at sensor startup.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>StoreInNVM</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum distance of far scan (near scan maximum&lt;/p&gt;&lt;p&gt;distance is set proportionally to half of the far scan).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>MaxDistanse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxDist">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sensor ID 0 â€“ 7&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Sensor_ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sensor_id">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configures the data output to clusters or objects.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>OutputType</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="output_type">
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>send objects</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>send clusters</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configures the transmitted radar power.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Radar Power</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="radar_power">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-3dB Tx gain</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-6dB Tx gain</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-9dB Tx gain</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selects the sorting index for the object list.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>SortIndex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sort_index">
        <item>
         <property name="text">
          <string>no sorting</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sorted by range</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sorted by RCS</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="CtrlRelay">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Relay control message (0x8) is sent if true and the&lt;/p&gt;&lt;p&gt;collision detection is activated&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>CtrlRelay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SendQuality">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cluster or object quality information (message 0x60C&lt;/p&gt;&lt;p&gt;or 0x702) is sent if true&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>SendQuality</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SendExInfo">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extended information (message 0x60D) is sent for&lt;/p&gt;&lt;p&gt;objects if true (if clusters are selected as output type&lt;/p&gt;&lt;p&gt;this value is ignored)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>SendExInfo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RCS_Threshold">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets the sensitivity of the cluster detection to standard or high sensitivity &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>RCS_Threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
